src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml

```diff
@@ -55,13 +54,122 @@
+                                    <ContentControl Grid.Row="1"
+                                                    Grid.Column="2"
+                                                    Content="{x:Bind ColorEntryAt(10), Mode=OneWay}"
+                                                    ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
+                                                    IsTabStop="False" />
+                                    <ContentControl Grid.Row="1"
+                                                    Grid.Column="3"
+                                                    Content="{x:Bind ColorEntryAt(11), Mode=OneWay}"
+                                                    ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
+                                                    IsTabStop="False" />
+                                    <ContentControl Grid.Row="1"
+                                                    Grid.Column="4"
+                                                    Content="{x:Bind ColorEntryAt(12), Mode=OneWay}"
+                                                    ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
+                                                    IsTabStop="False" />
+                                    <ContentControl Grid.Row="1"
+                                                    Grid.Column="5"
+                                                    Content="{x:Bind ColorEntryAt(13), Mode=OneWay}"
+                                                    ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
+                                                    IsTabStop="False" />
+                                    <ContentControl Grid.Row="1"
+                                                    Grid.Column="6"
+                                                    Content="{x:Bind ColorEntryAt(14), Mode=OneWay}"
+                                                    ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
+                                                    IsTabStop="False" />
+                                    <ContentControl Grid.Row="1"
+                                                    Grid.Column="7"
+                                                    Content="{x:Bind ColorEntryAt(15), Mode=OneWay}"
+                                                    ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
+                                                    IsTabStop="False" />
+                                    <TextBlock Grid.RowSpan="2"
+                                               Grid.Column="8"
+                                               Padding="10,0,10,0"
+                                               HorizontalAlignment="Center"
+                                               VerticalAlignment="Center"
+                                               AutomationProperties.AccessibilityView="Raw"

```

Comment:
Now that this is the only copy of the color scheme name in the UI, we need to do one of two things.

1. We need to make the entire tree `Raw` and set `AutomationProperties.Name="{x:Bind Name, Mode=OneWay}"` on the root element in the `DataTemplate` (actually, this might be the right way to go)

or

2. this can't be `Raw`

I saw tactic 1 in use in another application, but I can't remember which. It could be a really good way to ensure that a11y tools see the entire UI element as a single thing named "Campbell" or "One Half Dark". Maybe there's even a way to set the `AutomationProperties.ItemType` to `color scheme` or something. @carlos-zamora cc.

Reply:
The current implementation works as expected, actually. Looks like the list view is somehow applying the names to the list view item itself.

That said, the "default" tag isn't being read out. I don't think that's new though and I'm not sure how to fix it. You may need to bind the list view item like Dustin said in (1) above, but have the `"name"` be set to `"<name> <defaultTag (if present)>"`. Alas, that can probably be a follow-up.

Reply:
Relevant: https://github.com/microsoft/terminal/issues/13891